components/actions_menu/actions_menu_tutorial_tip.tsx

```diff
@@ -44,6 +44,7 @@ export const ActionsTutorialTip = ({handleOpen, handleDismiss, handleNext, showT
     return (
         <TourTip
             show={showTip}
+            onHidden={handleDismiss}
```

Comment:
@jfrerich This will create problem with other use cases,
I think would be better if use useEffect hook in the component 
```
  useEffect(() => {
        return () => {
            function here...
        };
    }, []);
    ```
    Have done it here 
    https://github.com/mattermost/mattermost-webapp/blob/175f8ebf5c03c0fae1ca0e808e32d39914c687b8/components/onboarding_tasks/complete_your_profile_tour_tip.tsx#L25

Reply:
@AshishDhama, I am going to push this out to another Jira ticket that will request this change and also the issue issue about the empty actions menu.  This way we an merge asap.